gulp/tasks/upload-preview.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+    'use strict';
+
+    var gulp = require('gulp'),
+        gitUtils = require('../util/gitUtils'),
+        childProcess = require('child_process'),
+        previewDomain = 'ux-test.edx.org',
+        previewSiteDir = '_preview_site';
+
+    gulp.task('upload-preview', function() {
+        var branch = gitUtils.currentBranch();
+        childProcess.execSync(
+            'aws s3 sync ' + previewSiteDir + ' s3://' + previewDomain + '/' + branch
+        );
+        console.log('Preview site ready at http://' + previewDomain + '/' + branch);
+    });
+}());
```

Comment:
@AlasdairSwan I thought that there was a convention of one file per task, but I'm happy to relax that if folks think it is clearer. @dan-f, what do you think?